### DISCUSSION TITLE
Support links in organization level workflow template descriptions

### DISCUSSION THREAD

**Discussion by NotMyFault** [community/community#12537] [Actions]
Heyo,

I created an organization level workflow template according to the [starter guide](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization), but links don't resolve in the `description` part of the `.properties.json` field.
https://github.com/NotMyFault-org/.github/blob/main/workflow-templates/jenkins-security-scan.properties.json#L3 is used to generate the action description but the outcome looks like
![](https://i.imgur.com/DD5cBbR.png)
You need to copy/paste the link to follow it.
Am I doing something wrong or are links not supported here?
Thanks in advance!

**Comment by ethomson** [comment]
Hi @NotMyFault - at the moment, links are not supported in this scenario. I'll talk with the marketplace team and make sure that this is on their backlog. Thanks much for the feedback.
